css/style.css

```diff
@@ -0,0 +1,1027 @@
+body {
+  margin-top: 0;
+  margin-bottom: 0;
+  margin-left: 0;
+  margin-right: 0;
+}
+
+ul.autor-time > li > a {
+  color: #aaa;
+}
+
+div.headerfirst-row {
+  background-color: #131313;
+  display: block;
+}
+
+ul.autor-time > li {
+  display: inline-block;
+  padding-left: 0;
+  color: black;
+}
+
+div.headerfirst-row > nav > ul > li,
+nav.wrapperleft-link,
+nav.wrapperright-link {
+  display: inline-block;
+}
+
+div.headerfirst-row nav ul > li > a {
+  text-decoration: none;
+  color: #bebebe;
+  padding-top: 0.2em;
+  padding-bottom: 0.2em;
+  padding-left: 0.5em;
+  padding-right: 0.5em;
+}
+
+div.headersecond-row nav ul > li a {
+  text-transform: uppercase;
+  color: #aaa;
+  text-decoration: none;
+  padding-top: 18px;
+  padding-bottom: 0;
+  padding-right: 10px;
+  padding-left: 10px;
+}
+
+footer div.group-link div ul > li > a {
+  color: #aaa;
+  margin-left: 3px;
+  margin-right: 3px;
+  text-decoration: none;
+}
+
+div.headerfirst-row nav ul > li > a:hover {
+  color: white;
+}
+
+footer div.group-link div > ul {
+  text-align: center;
+  margin-bottom: 0;
+  margin-top: 0;
+}
+
+div.headerfirst-row > nav.wrapperleft-link > ul {
+  padding-left: 17px;
+}
+
+div.headerfirst-row > nav.wrapperright-link {
+  float: right;
+}
+
+div.headersecond-row nav ul > li a:hover {
+  color: black;
+  text-transform: uppercase;
+  text-decoration: none;
+  padding-top: 18px;
+  padding-bottom: 0;
+  padding-right: 10px;
+  padding-left: 10px;
+}
+
+header > div.headersecond-row > nav,
+header > div,
+div.headersecond-row nav ul > li {
+  display: inline-block;
+}
+
+div.headersecond-row {
+  display: block;
+  height: 3em;
+}
+
+div.headersecond-row > div {
+  float: right;
+  display: flex;
+  margin-right: 1em;
+  padding-left: 1em;
+  border-left: 1px solid #aaa;
+}
+
+div.yellow-bar {
+  background-color: rgb(255, 221, 51);
+}
+
+div.header-lastrow {
+  display: block;
+  padding-top: 0.5em;
+  padding-bottom: 0.5em;
+}
+
+div.header-lastrow > span {
+  margin-left: 1.5em;
+}
+
+@media (max-width: 935px) {
+  nav.wrapperleft-link > ul {
+    display: none;
+  }
+
+  nav.wrapperright-link > ul {
+    display: none;
+  }
+
+  div.headersecond-row > div > p,
+  div.headersecond-row > nav > ul > li:nth-child(2),
+  div.headersecond-row > nav > ul > li:nth-child(3),
+  div.headersecond-row > nav > ul > li:nth-child(4),
+  div.headersecond-row > nav > ul > li:nth-child(5),
+  div.headersecond-row > nav > ul > li:nth-child(6),
+  div.headersecond-row > nav > ul > li:nth-child(7),
+  div.headersecond-row > nav > ul > li:nth-child(8) {
+    display: none;
+  }
```

Comment:
We have to remove the repeated code and follow the DRY principle. Instead of using such long selector (  div.headersecond-row > nav > ul > li:nth-child(2),), we can define class with the required styling and add where ever we need. Another option is we can we `not` selector. checkout this [article](https://speckyboy.com/good-bad-css-practices)

Reply:
I modified this part.